src/test/java/org/folio/rest/impl/FeeFineActionsAPITest.java

```diff
@@ -247,7 +247,8 @@ private JsonObject createNoticeContext(String feeFineType,
         .put("actionAmount", amountAction)
         .put("actionDateTime", dateAction)
         .put("balance", balance)
-        .put("actionAdditionalInfo", ""));
+        .put("actionAdditionalInfo", "")
```

Comment:
Why not add a non-empty value to `actionAdditionalInfo`?

Reply:
Added coment